common/presenters/events-to-timeline/event-details-helpers.test.js

```diff
@@ -0,0 +1,251 @@
+const proxyquire = require('proxyquire')
+
+const i18n = require('../../../config/i18n')
+
+const {
+  getMoveDetails,
+  getEventAgency,
+  getEventContext,
+  getEventRelationships,
+  getEventableDetails,
+  getEventProperties,
+} = proxyquire('./event-details-helpers', {
+  '../../../common/lib/api-client/models': {
+    event: {
+      fields: {
+        event_type: '',
+        prop_a: '',
+        prop_b: '',
+        details: '',
+        eventable: {
+          jsonApi: 'hasOne',
+        },
+        rel_a: {
+          jsonApi: 'hasOne',
+          type: 'rel_as',
+        },
+        rel_b: {
+          jsonApi: 'hasOne',
+          type: 'rel_bs',
+        },
+      },
+    },
+  },
+})
+
+describe('Timeline events', function () {
+  describe('Event details helpers', function () {
+    beforeEach(function () {
+      sinon.stub(i18n, 't').returnsArg(0)
+      sinon.stub(i18n, 'exists').returns(false)
+    })
+
+    context('#getMoveDetails', function () {
```

Comment:
I know its syntactic sugar, but should these be `describes` instead of `context`? I think that's how it is in the rest of the codebase?

Reply:
Erm, it's actually a bit all over the place. If only we had a linter that enforced a convention...   

(Will change these instances if you confirm ;) )

Reply:
Didn't bother to wait. Done